app/main.py

```diff
@@ -1,25 +1,103 @@
+from __future__ import annotations
 from abc import ABC
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(
+            self,
+            owner: SlideLimitationValidator,
+            name: str
+    ) -> None:
+        self.public_name = name
+        self.protected_name = "_" + name
+
+    def __get__(
+            self,
+            instance: SlideLimitationValidator,
+            owner: SlideLimitationValidator
+    ) -> None:
+        return getattr(instance, self.protected_name)
+
+    def __set__(
+            self,
+            instance: SlideLimitationValidator,
+            value: int
+    ) -> None:
+        if self.min_amount <= value <= self.max_amount:
+            return setattr(instance, self.protected_name, value)
+        # else:
+        #     raise ValueError(
+        #         f"{self.public_name} should be >= "
+        #         f"{self.min_amount} and <= {self.max_amount}")
```

Comment:
Raise ValueError but without using variables in f-strings. Then tests should go ok